vertical-pod-autoscaler/README.md

```diff
@@ -151,6 +151,27 @@ or recreated by their controller due to Updater's activity).
 
 More on the architecture can be found [HERE](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/autoscaling/vertical-pod-autoscaler.md).
 
+### Tear down
+
+Note that if you stop running VPA in your cluster, the resource requests
+for the pods already modified by VPA will not change, but any new pods
+will get resources as defined in your controllers (i.e. deployment or
+replicaset) and not according to previous recommendations made by VPA.
+
+To stop using Vertical Pod Autoscaling in your cluster:
+* Remove any verticalpodautoscaler objects you created. For example:
```

Comment:
I think this step shouldn't be necessary if you do ./hack/vpa-down.sh, which deletes the CRD and so removes all VPA objects.

Reply:
Done